include/libopencm3/stm32/l0/rcc.h

```diff
@@ -505,32 +507,40 @@ enum rcc_periph_clken {
 	RCC_TIM21 = _REG_BIT(0x34, 2),
 	RCC_TIM22 = _REG_BIT(0x34, 5),
 	RCC_MIFI = _REG_BIT(0x34, 7),
+	RCC_FW = _REG_BIT(0x34, 7),
```

Comment:
heh, then they've changed it in a newer rev :(  My copy of rm0367 and rm0377 both call the bit MIFI  In that case we should just make a single commit dropping MIFI and replacing it with FW.